src/styles.css/headerFoto.css

```diff
@@ -0,0 +1,39 @@
+.card {
```

Comment:
name file card.css

Reply:
bugs fixed, tests passed. Of course, after landing tasks, I would delete half of it here)